app/templates/index.hbs

```diff
@@ -70,7 +70,7 @@
 
                     {{#if (eq fragmentField.type 'Date')}}
                       <p class='my1 cf-card-field'>
-                        {{fragmentField.key}}: {{if fragmentField.value (moment-format fragmentField.value 'dddd, Do MMMM YYYY')}}
+                        {{fragmentField.key}}: {{if fragmentField.value (moment-format (utc fragmentField.value) 'dddd, Do MMMM YYYY')}}
```

Comment:
This is the solution because the Pikaday datepicker selector is returning the date in UTC format
```
{{pikaday-input
    class='cf-form-input'
    format="dddd, Do MMMM YYYY"
    useUTC=true
    value=fragmentField.value
    onSelection=(action 'saveDate' fragmentField)
    position='bottom left'
    reposition=false
}}
```